TrackletAlgorithm/InputRouterTop.h

```diff
@@ -1,25 +0,0 @@
-#ifndef TrackletAlgorithm_InputRouterTop_h
-#define TrackletAlgorithm_InputRouterTop_h
-
-#include "InputRouter.h"
-
-
-// at the moment I'm changing this for each link 
-// is that really the only way to do this? 
-
-// only for PS10G_3_A for now 
-// other cases to be added 
-// when I have final link map
-//constexpr unsigned int cTest = (const)(std::atoi(getenv ("TEST")));
-constexpr unsigned int cNMemories = 7;
```

Comment:
@sseifeln The C/RTL cosimulation currently crashes with a seg-fault, and the reason seems to be that this constant has changed. Looking in the InputRouter_parameters.h that is generated by generate_IR.py, I see that now
```c++
constexpr unsigned int cNMemories_IR_DTC_PS10G_3_A = 14;
```
Manually setting this back to 7 allows the cosimulation to run and pass.

Any idea why this constant changed?

Reply:
Yeah I see where the mistake is now (not looking for an exact match when calculating the constant in the script) - just trying to fix it and will commit